### DISCUSSION TITLE
Microsoft accounts will be suported?

### DISCUSSION THREAD

**Discussion by msmanui** [games647/FastLogin#600] [General]
Hi there.

Migration of accounts has started. If a player Mojang migrates to Microsoft, will remain the same uuid? 
This will afect the login premium with FastLogin?

**Comment by TuxCoding** [comment]
Yes, it will stay the same. They only change the login to your account not the process on joining the server. Otherwise older server would no longer be supported.

https://github.com/games647/FastLogin/wiki/FAQ#is-this-plugin-compatible-with-microsoft-accounts

**Reply by msmanui** [reply]
> Yes, it will stay the same. They only change the login to your account not the process on joining the server. Otherwise older server would no longer be supported.
> 
> https://github.com/games647/FastLogin/wiki/FAQ#is-this-plugin-compatible-with-microsoft-accounts

Ok thanks for you answer.
I have another doubt: ¿If a player who has enabled the premium feature and changes his nickname is there anyway to save his progress even he changes of username in his minecraft account?

**Reply by TuxCoding** [reply]
yes, if premiumUUID is enabled and plugins use this id, then the data stays the same. The UUID is meant to be unique non-changeable provided by Mojang.

**Reply by msmanui** [reply]
> yes, if premiumUUID is enabled and plugins use this id, then the data stays the same. The UUID is meant to be unique non-changeable provided by Mojang.

Ok i see! Im reading the functionallity but cant understand well. 
I currently using FastLogin with /premium to enable for users. I have a few premiums account register with alternative UUID. What will happen if i enbable the premium uuid for those users? And quite didnt understand the conversion of OFFLINE UUID to the premium uuid, how would i do that? 

Basically, how can i enable the premium uuid feature with nameChangeCheck and without losing their inventories, data , etc and prevent any further issue like the /ban stuff

**Reply by TuxCoding** [reply]
That's not possible with FastLogin. However there are third party software that can do the conversion for you.

**Comment by msmanui** [comment]
Which software? Can u recommend me one please?


El 18 ago. 2021, a la(s) 05:42, games647 ***@***.***> escribió:

﻿

That's not possible with FastLogin. However there are third party software that can do the conversion for you.
